frontend/src/Pages/Dashboardui.js

```diff
@@ -0,0 +1,193 @@
+import React, { useState, useEffect } from 'react';
+import { styled, createTheme, ThemeProvider } from '@mui/material/styles';
+import CssBaseline from '@mui/material/CssBaseline';
```

Comment:
to reduce imports `import { CssBaseline, Button } from '@mui/material';` you can use as.